README.rst

```diff
@@ -145,6 +145,21 @@ PUT JSON data:
              (lambda (&key data &allow-other-keys)
                (message "I sent: %S" (assoc-default 'json data)))))
 
+PUT JSON data including non-ascii strings:
+
+.. code:: emacs-lisp
+
+  (setq request-conding-system 'utf-8)
```

Comment:
I see it's consistent everywhere.
Still shouldn't it be `coding` instead of `conding`?

```suggestion
  (setq request-coding-system 'utf-8)
```

Reply:
I fix this.